apps/fluent-tester/src/FluentTester/TestComponents/Common/styles.ts

```diff
@@ -131,3 +131,16 @@ export const separatorStackStyle: IStackProps['style'] = {
   flexDirection: 'row',
   justifyContent: 'space-evenly',
 };
+
+export const mobileStyles = StyleSheet.create({
+  testListItems: {
+    width: '100%',
+    height: 'auto',
+    fontSize: 18,
+    paddingVertical: 8,
+  },
+  testListSeparator: {
+    borderColor: '#e1e1e1',
```

Comment:
nit: I would rather not place this value (gray100) here and rather have it specified via tokens.

Reply:
added to themedStyles and set the color to refer menuDivider